package Tasks.Exception;

public class ExceptionUsingThrows {
    public static void main(String[] args)  throws Exception{

        int a =0;
//        int b= 10/a;
        String ab= null;
        String ba= ab.trim();
//        System.out.println(Exception.class);
//        System.out.println(b);
    }
}
